### ISSUE TITLE
[DragonRealm] [Chromaticraft V32c] Casting Table not accepting energy

### ISSUE THREAD

**Issue by tprussak** [ReikaKalseki/Reika_Mods_Issues#3252]
I am playing on the latest version of Dragon Realm as of 7/21/23, on hard configs. I am trying to craft a crystal furnace, but the casting table will not accept energy. According to the examination crystals, it is connected to Portokali, Zambarau, and Tahara pylons, but it does not draw any power. Do I need to connect every pylon type before I can do any pylon casting? I do not recall needing to do that on older versions.

Casting table GUI
![image](https://github.com/ReikaKalseki/Reika_Mods_Issues/assets/1276028/13c034bc-3b1d-4831-aa9c-8de72d8ed4ca)

Examination Crystals:
![image](https://github.com/ReikaKalseki/Reika_Mods_Issues/assets/1276028/22855357-0e60-41c9-8adc-d92422c37ef3)

**Comment by tprussak** [comment]
I also tried making a Zambarau Potion crystal, with no success. I confirmed with the ritual altar that I can pull both Zambarau and Tahara energy. I also checked in a previous version (v31b in Revolution 3) and it does seem to work there without issues, with only some pylons connected.

**Comment by DrunkenSquirrel** [comment]
I ran into the same issue and looking at the source code there's a bug (or at least I believe it to be a bug) in the calculation for the throughput calculation for the casting table. New in v32 you're supposed to get a bonus for grouping the transmitters by group (the ones you use to craft a crystal core) on each side, however without any groups the throughput bonus is 0 which when multiplied by the base throughput results in 0, making the table unable to accept any lumen energy.

tl:dr: Put the casting temple repeaters in groups to fix

**Comment by bakaawful** [comment]
Thank you. I had the same issue, and grouping the repeaters by Crystal Group around my casting table did fix it.

In case anybody needs further explanation: the 16 repeaters you build into your Casting Complex need to have their type (determined by the Crystal Rune just under the Crystal Repeater block) grouped up, by type, as the Crystal Shards are grouped in the four Crystal Groups. 
Your casting Complex is a square (if you view it from a top-down view), and it has four sides, each side with four Crystal Repeaters.
One side should have these Crystal Repeater types: Karmir, Nila, Kurauri, Zambarau.
The next: Vadali, Kitrino, Asveste, Kijani.
The next: Ruzova, Ruskea, Galazio, Portokali.
The next: Kuro, Argia, Ykri, Tahara.

The order within a side doesn't matter. (I adjusted it to allow the Casting Delegate to receive energy of appropriate colours, and the Casting Table kept receiving power, no issues.)

Hope it saves you a look-up in the Crystal Groups.

**Comment by ReikaKalseki** [comment]
This is fixed.
